chain-tx-enclave-next/tx-query-next/app-runner/src/zmq_helper.rs

```diff
@@ -0,0 +1,189 @@
+use std::{
+    convert::TryInto,
+    future::Future,
+    pin::Pin,
+    task::{Context, Poll},
+    vec::IntoIter,
+};
+
+use async_zmq::{request, Message, MultipartIter, Request};
```

Comment:
what's a reason for `async_zmq` ? Existing zmq bridging used synchronous REQ-REP which still be wrapped?

Reply:
`async-zmq` hooks `zmq`'s notification file descriptors with `tokio` runtime to get async IO.

Reply:
where does that make a difference? that `inner.socket.send` / `unner.socket.recv`won't block?

Reply:
Yes. They won't block. They are `async` functions (https://docs.rs/async_zmq/0.3.2/async_zmq/request/struct.Request.html).